src/plans/plan.jl

```diff
@@ -35,6 +37,57 @@ function get_manopt_parameter(f, e::Symbol, args...)
 end
 get_manopt_parameter(f, args...) = nothing
 
+"""
+    get_manopt_parameter(element::Symbol; default=nothing)
+
+Access global [`Manopt`](@ref) parametersadressed by a symbol `element`.
+We first dispatch on the value of `element`.
+
+if the value is not set, `default` is returned
+
+The parameters are queried from the global settings using [`Preferences.jl`](https://github.com/JuliaPackaging/Preferences.jl),
+so they are persistent within your activated Environment.
+
+# Currenlty used settings
+
+`:Mode`
+the mode can be set to `"Tutorial"` to get several hints especially in scenarios, where
+the optimisation on manifolds is different from the usual “experience” in
+(classicall, Euclidean) optimization.
+Any other value has the same effect as not setting it.
+"""
+function get_manopt_parameter(
+    e::Symbol, args...; default=get_manopt_parameter(Val(e), Val(:default))
+)
+    return @load_preference("$(e)", default)
+end
+# Handle empty defaults
+get_manopt_parameter(::Symbol, ::Val{:default}) = nothing
+get_manopt_parameter(::Val{:Mode}, v::Val{:default}) = ""
+
+"""
+    set_manopt_parameter!(element::Symbol, value::Union{String,Bool})
+
+Set global [`Manopt`](@ref) parameters adressed by a symbol `element`.
+We first dispatch on the value of `element`.
+
+The parameters are stored to the global settings using [`Preferences.jl`](https://github.com/JuliaPackaging/Preferences.jl).
+
+Passing a `value` of `""` deletes an entry from the preferences.
+Whenever the `LocalPreferences.toml` is modified, this is also `@info`rmed about.
+"""
+function set_manopt_parameter!(e::Symbol, value::Union{String,Bool,<:Number})
+    if length(value) == 0
+        @delete_preferences!("$(e)")
+        v = get_manopt_parameter(e, Val(:default))
+        default = isnothing(v) ? "" : (length(v) == 0 ? "" : " ($(get_manopt_parameter))")
```

Comment:
Is this really supposed to just print `get_manopt_parameter` without any arguments?

Reply:
I wanted to avoid to have printed `"nothing"` in the string. That is what this avoids.

Reply:
ah and if the value is length 0, eg for a string, this also avoids () and does not add brackets, like the last case does.

Reply:
I see. Though maybe it would be better to print strings in `"`, so that empty string is `""`?

Reply:
Hm, maybe, maybe we are also just overcomplicating things. In the very most cases this indeed will only be strings and `resetting to default ""` also does not read so nice.